### PR TITLE
Rule test to branch

### PR DESCRIPTION
**Pull request by marceloarb**

### CODE REVIEW

**Comment by dbrewster42** [comment]
This is the worst code I have ever seen. Absolutely terrible

File: src/main/java/com/teksystems/Capstone3BackEnd/Models/fff.java [hunk 1]
```diff
@@ -1,7 +0,0 @@
-package com.teksystems.Capstone3BackEnd.Models;
-
-public class fff {
```
Comment (ghost) [inline src/main/java/com/teksystems/Capstone3BackEnd/Models/fff.java:3]: This is a beautiful class.  I think it stays.

**Comment by jbozarov** [comment]
This is a review test

File: src/main/java/com/teksystems/Capstone3BackEnd/Models/fff.java (hunk 1 above)
Comment (marceloarb) [reply src/main/java/com/teksystems/Capstone3BackEnd/Models/fff.java:3]: No, that is gone.

**Review by ghost — APPROVED** [review]

**Review by dbrewster42 — COMMENTED** [review]
adfs

**Review by jj-moreno — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Approved

**Review by jbozarov — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**